src/widgets/FileWidget/index.js

```diff
@@ -142,10 +153,22 @@ const getProps = (props) => {
     baseUiSchemaPath = 'items';
     pathUiSchemaPath = 'items';
     nameUiSchemaPath = 'items';
+    adjustedRemoveComponent = (
+      adjustedRemoveComponent
+      || (uiSchema['ui:options'] && uiSchema['ui:options'].RemoveComponent)
+    );
     adjustedUploadComponent = (
       adjustedUploadComponent
       || (uiSchema['ui:options'] && uiSchema['ui:options'].UploadComponent)
     );
+    adjustedRemoveStyle = (
+      adjustedRemoveStyle
+      || (uiSchema['ui:options'] && uiSchema['ui:options'].removeStyle)
+    );
+    adjustedRemoveLabel = (
+      adjustedRemoveLabel
+      || (uiSchema['ui:options'] && uiSchema['ui:options'].removeLabel)
+    );
```

Comment:
Please move the `adjustedUploadComponent` here so it's grouped with the other `adjustedUploadProps`.